hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/server/HttpFSAuthenticationFilter.java

```diff
@@ -69,27 +72,11 @@
   @Override
   protected Properties getConfiguration(String configPrefix,
       FilterConfig filterConfig) throws ServletException{
-    Properties props = new Properties();
+    System.out.println("getConfiguration1");
     Configuration conf = HttpFSServerWebApp.get().getConfig();
-
-    props.setProperty(AuthenticationFilter.COOKIE_PATH, "/");
```

Comment:
Have you intentionally deleted this line?
```
props.setProperty(AuthenticationFilter.COOKIE_PATH, "/");
```

Reply:
Yes, the HttpServer2.getFilterProperties() will end up doing this.

BTW the order of the overwriting has changed as I mentioned in the commit message. Also here the line 80 contained an error, calling the getValue() did not substituted the variable, but on the line 88 it did.